### DISCUSSION TITLE
service ca operator

### DISCUSSION THREAD

**Discussion by stu93303148** [okd-project/okd#824] [Q&A]
I don't know why only some nodes get CA information? Why not all nodes get CA?

![1111](https://user-images.githubusercontent.com/1113604/130389988-c8ffbae0-e384-4f71-9eeb-c053013d2842.png)

**Comment by rvanderp3** [comment]
Hi @stu93303148 I have a couple of questions:
- By CA Information, are you referring to the ingress certificate?
- How many replicas of the router are you running?
